src/net/db.rs

```diff
@@ -112,9 +112,17 @@ impl fmt::Debug for LocalPeer {
 }
 
 impl LocalPeer {
-    pub fn new(network_id: u32, parent_network_id: u32, port: u16, key_expire: u64, data_url: UrlString) -> LocalPeer {
+    pub fn new(network_id: u32, parent_network_id: u32, port: u16, key_expire: u64, data_url: UrlString, seed: Vec<u8>) -> LocalPeer {
+
+        let mut re_hashed_seed = seed;
```

Comment:
Why not just provide the private key?

Reply:
It was insanely straightforward :).
You prefer a private key? hex? wif?

Reply:
Don't worry about it; I've already changed it in my local copy.  It'll just take the private key, and the testnet node will generate it from its seed when instantiating the peer DB.

Reply:
gotcha, I thought you wanted to pass the private key in the config, instead of a seed.